projects/animation-maker/js/maker.js

```
@@ -290,13 +290,13 @@ async function init() {
 function setupTheme() {
     const urlParams = new URLSearchParams(window.location.search);
     const themeParam = urlParams.get('theme');
-    if (themeParam && (themeParam === 'light' || themeParam === 'dark')) {
-        localStorage.setItem('maker-theme', themeParam);
-    }
-
     const savedTheme = localStorage.getItem('maker-theme') || localStorage.getItem('studio-theme');
     const prefersDark = window.matchMedia('(prefers-color-scheme: dark)').matches;
-    state.currentTheme = savedTheme || (prefersDark ? 'dark' : 'light');
+
+    state.currentTheme = (themeParam && (themeParam === 'light' || themeParam === 'dark'))
```

ESLint

[error] Replace `·(themeParam·&&·(themeParam·===·'light'·||·themeParam·===·'dark')` with `⏎········themeParam·&&·(themeParam·===·'light'·||·themeParam·===·'dark'`

(prettier/prettier)